### PR TITLE
[Merged by Bors] - chore: make `SimplexCategory.skeletalFunctor` a functor to NonemptyFinLinOrdCat.{0}

### PR DESCRIPTION
**Pull request by joelriou**
`SimplexCategory` used to have a universe parameter, but for already some time, it is no longer the case. However, the functor `SimplexCategory.skeletalFunctor` was still a functor from the simplex category to `NonemptyFinLinOrdCat.{v}`. This PR changes this to `NonemptyFinLinOrdCat.{0}`. The main consequence of this is that if `C` is a category, the `n`-simplices of the simplicial set `nerve C` (see `AlgebraicTopology.Nerve`) are now definitionally equal to `Fin (n+1) ⥤ C`. 

---
(Previously, there was an annoying `ULift.{0,0}` when we unfolded the definition of the nerve.)

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
Can you move the PR description above the `---` so that it appears in the commit message?

File: Mathlib/AlgebraicTopology/SimplexCategory.lean [hunk 1]
```diff
@@ -352,13 +352,13 @@ section Skeleton
 /-- The functor that exhibits `SimplexCategory` as skeleton
 of `NonemptyFinLinOrd` -/
 @[simps obj map]
-def skeletalFunctor : SimplexCategory ⥤ NonemptyFinLinOrd.{v} where
-  obj a := NonemptyFinLinOrd.of <| ULift (Fin (a.len + 1))
-  map f := ⟨fun i => ULift.up (f.toOrderHom i.down), fun i j h => f.toOrderHom.monotone h⟩
+def skeletalFunctor : SimplexCategory ⥤ NonemptyFinLinOrd where
+  obj a := NonemptyFinLinOrd.of (Fin (a.len + 1))
```
Comment (eric-wieser) [inline Mathlib/AlgebraicTopology/SimplexCategory.lean:356]: Is the argument here that if you want `NonemptyFinLinOrd.{v}`, you cam compose with something like `NonemptyFinLinOrd.ulift.{v} : NonemptyFinLinOrd.{u} ⥤ .NonemptyFinLinOrd {max u v}`?

File: Mathlib/AlgebraicTopology/SimplexCategory.lean [hunk 2]
```diff
@@ -367,24 +367,20 @@ theorem skeletal : Skeletal SimplexCategory := fun X Y ⟨I⟩ => by
     ext
     simpa
   apply Fintype.card_congr
-  exact Equiv.ulift.symm.trans
-    (((skeletalFunctor.{0} ⋙ forget NonemptyFinLinOrd).mapIso I).toEquiv.trans Equiv.ulift)
+  exact (((skeletalFunctor ⋙ forget NonemptyFinLinOrd).mapIso I).toEquiv)
```
Comment (eric-wieser) [inline Mathlib/AlgebraicTopology/SimplexCategory.lean:370]:
```suggestion
  exact ((skeletalFunctor ⋙ forget NonemptyFinLinOrd).mapIso I).toEquiv
```

**Review by eric-wieser — APPROVED** [review]
bors d+

File: Mathlib/AlgebraicTopology/SimplexCategory.lean (hunk 1 above)
Comment (joelriou) [reply Mathlib/AlgebraicTopology/SimplexCategory.lean:356]: Yes.

**Comment by joelriou** [comment]
Thanks!

bors merge

**Pull request closed without merging**